### ISSUE TITLE
DemoCards.jl integration

### ISSUE THREAD

**Issue by asinghvi17** [LuxDL/DocumenterVitepress.jl#137]
Things that need to be done for this:
- Support for `assets` to load custom CSS
- A custom theme for DemoCards, which:
	- Uses a version of @lazarusA's vue component in BeautifulMakie
	- Allows and emits SEO tags as shown in [the Vitepress docs](https://vitepress.dev/reference/frontmatter-config#head)
	- Displays the card in the way BeautifulMakie does (cover image/video on top)
	- Has description text at the bottom or something

**Comment by lazarusA** [comment]
like these ones? 

https://lux.csail.mit.edu/stable/tutorials/

**Comment by asinghvi17** [comment]
Yes!  We might want to change the aspect ratio to the default Makie theme though

**Comment by asinghvi17** [comment]
Looks like it's manually generated, but that gives us a good starting point for the demo card format!!
https://github.com/LuxDL/Lux.jl/blob/main/docs/src/tutorials/index.md?plain=1

**Comment by lazarusA** [comment]
and is using the same Gallery component from BMakie 😄 .

**Comment by avik-pal** [comment]
> and is using the same Gallery component from BMakie 😄 .

That + CodeGPT :sweat_smile:

**Comment by asinghvi17** [comment]
I guess I already did this in https://github.com/asinghvi17/OhMyCards.jl, just needs some love now :D
